### PR TITLE
fixes for the fixes

### PR DESCRIPTION
**Pull request by RisingGlitch**
fixes custom themes being created with a "," always, which causes them to not show at all.

### CODE REVIEW

File: static/assets/scripts/s.js [hunk 1]
```diff
@@ -284,7 +284,7 @@ function newCustomTheme() {
     alert("Name cannot contain a comma.")
     return
   }
-  if (!localStorage.getItem("customThemes")) {
+  if (localStorage.getItem("customThemes" !== "")) {
```
Comment (IncognitoTGT) [inline static/assets/scripts/s.js:287]: kek you could've just removed the ! from before

Comment (RisingGlitch) [reply static/assets/scripts/s.js:287]: im so confused rn

Comment (IncognitoTGT) [reply static/assets/scripts/s.js:287]: ` if (localStorage.getItem("customThemes")) {`

do that

Comment (RisingGlitch) [reply static/assets/scripts/s.js]: I got it

**Review by IncognitoTGT — APPROVED** [review]
balls

**Pull request merged**